web/src/search_pill.ts

```diff
@@ -182,8 +191,10 @@ export function generate_pills_html(suggestion: Suggestion): string {
         // text and no pill.
         if (pill.type === "search_non_user" && pill.operator !== "search") {
             description_html = suggestion.description_html;
+        } else if (pill.type === "search_user" && is_sent_by_me_pill(pill)) {
+            description_html = "Messages you sent";
```

Comment:
Does this correctly handle `-sender:me`?

Reply:
No, and it also isn't translated. I'll fix both of those.